### PR TITLE
RSWEB-7179: style(nav): increase font size for links

### PR DESCRIPTION
**Pull request by runrog**
Increased the font size on drop down links in global nav and increased font weight/size on sub navs to match global nav
screen shot 2016-10-04 at 1 10 14 pm

### CODE REVIEW

File: styleguide/_themes/derek/scss/components/subnav.scss [hunk 1]
```diff
@@ -74,7 +74,8 @@
 
 .subnav-link {
   color: $white;
-  font-weight: 300;
+  font-size: 14px;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/components/subnav.scss]: Can we use `rem` or `em` here?

Comment (runrog) [inline styleguide/_themes/derek/scss/components/subnav.scss]: @WidgetsBurritos switched

**Comment by WidgetsBurritos** [comment]
@amberRrucker this needs a design review.

**Comment by amberRrucker** [comment]
screen shot 2016-10-04 at 1 43 49 pm
screen shot 2016-10-04 at 1 44 13 pm
screen shot 2016-10-04 at 1 44 24 pm
screen shot 2016-10-04 at 1 44 33 pm

Theres so funky stuff going on in tablet on hover of the dropdowns. Overlapping, crazy sideways stuff, losing the hover ect. Not sure how much is related. Also do we care about the browser clipping the side of the text in the small end of the large viewport?

**Comment by runrog** [comment]
@amberRrucker deattached sub menu is fixed, had to adjust font size in tablet. The cutoff issues however were already present.

**Comment by amberRrucker** [comment]
Since there is a bug ticket in for the hover state loss in submenus and the overlap on tertiary menus Ill give it a +1

**Pull request merged**